tsLQC/autots_hyperparameter_tuning.py

```diff
@@ -0,0 +1,46 @@
+import itertools
+from autots import AutoTS
+
+from template_generation import templateGeneration
+from backtesting_using_growthrate import backtesting
+
+df = templateGeneration()
+
+
+def hyperparameter_tuning(ts, backtest_length):
+    print("----------------------------------Hyperparameter Tuning ------------------------------------")
+    hyperparams_dict = {}
+    params = {'validation_points': [4, 6, 8, 10], 'validation_method': ['backward', 'even', 'similarity']}
+
+    for validation_points, validation_method in itertools.product(*params.values()):
+
+        try:
+            print(validation_points, validation_method)
+
+            model = AutoTS(forecast_length=validation_points,
+                           frequency='infer',
```

Comment:
get the constant variables into constants or some kind of setting

Reply:
moved all constants to constant.py